fedora-live-image-build.sh

```diff
@@ -23,6 +23,13 @@ TESTTYPE="packaging skip-on-rhel"
 
 . ${KSTESTDIR}/functions.sh
 
+prepare_disks() {
+    tmpdir=$1
+
+    qemu-img create -q -f qcow2 ${tmpdir}/disk-a.img 12G
```

Comment:
AFAIK, `qemu-img` doesn't allocate the specified image size on the disk when creating it. Because of that I think the disk size should be defined even bigger (20G?), just to prevent having to increase the value in the near future again. I do not expect the size of the image will stop increasing :-)

Reply:
I agree, let's take a benefit of the qcow2 format.

Reply:
Updated